src/reynir_correct/wrappers.py

```diff
@@ -196,7 +199,7 @@ def check_errors(**options: Any) -> str:
 def check_spelling(**options: Any) -> str:
     # Initialize sentence accumulator list
     # Function to convert a token list to output text
-    format = options.get("format", "")
+    format = options.get("format", "json")
```

Comment:
Is "json" a better default than "text"? It may well be, but worth asking.

Reply:
My thinking was that the API returns json, but text might also be suitable